Description: Planner is unable to read files from command line.
 This patch solves the issue.
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=656373#c8
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=656373
Author: Wences Arana <aranax@debian.org.gt>
Last-Update: 2011-12-10

--- a/src/planner-main.c
+++ b/src/planner-main.c
@@ -33,11 +33,9 @@
 
 /* Command line options */
 static gchar *geometry = NULL;
-static gchar **args_remaining = NULL;
 
 static GOptionEntry options[] = {
 		{ "geometry", 'g', 0, G_OPTION_ARG_STRING, &geometry, N_("Create the initial window with the given geometry."), N_("GEOMETRY")},
-		{ G_OPTION_REMAINING, 0, 0, G_OPTION_ARG_FILENAME_ARRAY, &args_remaining, NULL, N_("FILES") },
 		{ NULL }
 	};
 
